roles/ansible-tdp-common-actions/tasks/config-chrony-for-vms.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Configure chrony for VM hibernation/reboot
+  replace:
+    dest: /etc/chrony.conf
+    regexp: '^makestep(.*)$'
```

Comment:
why `(...)` ? you are not doing any pattern matching to inject some value inside of it.

Reply:
Mistake on my part. They're removed now.